Jenkinsfile

```diff
@@ -32,27 +32,32 @@ def build(Build build) {
     node(build.worker.label) {
       try {
         echo "Worker: ${env.NODE_NAME}"
+        gitNotify ("New CI: " + build.name, "Started...", 'PENDING')
```

Comment:
Why is that "New"?

Reply:
We decided for some period use 2 CI: this CI(new) and current one(old) 
Old CI generate: `continuous-integration/jenkins/pr-head` check name.
New CI generate `New CI: <info>` check names.
'New CI' prefix needs to not scare developers if build marked as failed with new ci, because "new" can be not stable. 
After Old CI will be delete, the 'New CI' prefix will be rename into  'Jenkins'